roqoqo-mock/tests/integration/backend.rs

```diff
@@ -34,9 +34,6 @@ fn run_simple_circuit_measure_qubit() {
     assert_eq!(out_reg.len(), 1);
     for reg in out_reg.iter() {
         assert_eq!(reg.len(), 1);
-        for val in reg.iter() {
-            assert!(*val || !*val);
-        }
```

Comment:
How come this was removed?

Reply:
The same applies below

Reply:
Clippy was giving a warning and since it seems that this test is not doing anything I thought we should just remove it. It looks like it asserts that the value is either true or false